flutter/lib/src/navigation/time_to_display_tracker.dart

```diff
@@ -0,0 +1,92 @@
+import 'dart:async';
+
+import 'package:meta/meta.dart';
+import '../integrations/integrations.dart';
+
+import '../../sentry_flutter.dart';
+import '../native/sentry_native.dart';
+import 'time_to_display_transaction_handler.dart';
+import 'time_to_initial_display_tracker.dart';
+
+@internal
+class TimeToDisplayTracker {
+  final SentryNative? _native;
+  final TimeToDisplayTransactionHandler _ttdTransactionHandler;
+  final TimeToInitialDisplayTracker _ttidTracker;
+  final bool _enableTimeToFullDisplayTracing;
+
+  TimeToDisplayTracker({
+    required bool enableTimeToFullDisplayTracing,
+    required TimeToDisplayTransactionHandler ttdTransactionHandler,
+    TimeToInitialDisplayTracker? ttidTracker,
+  })  : _native = SentryFlutter.native,
+        _enableTimeToFullDisplayTracing = enableTimeToFullDisplayTracing,
+        _ttdTransactionHandler = ttdTransactionHandler,
+        _ttidTracker = ttidTracker ?? TimeToInitialDisplayTracker();
```

Comment:
we should always set the `isManual` to false when a route is pushed.
If the user use the `SentryDisplayWidget` only in one route, the others will never complete, as we use a singleton that thinks all routes are manual.
If we do it here in `startTracking`, let's make sure the manual API works as expected

Reply:
yeah, just noticed it, forget about it 😄